tna/insights/management/commands/fetch_insights_pages.py

```diff
@@ -55,6 +58,10 @@ def fetch_page_data(url):
 class Command(BaseCommand):
```

Comment:
This and the other command have so much in common - normally I would suggest we create a class for both commands to extend from. However, given they might not be used for much longer, that might not be worth it.

Reply:
Agree.